src/blocklypropclient.js

```diff
@@ -387,17 +356,13 @@ function establish_socket() {
 
         // Log errors
         connection.onerror = function (error) {
-            console.log('WebSocket Error');
-            console.log(error);
-
-            //$("#client-searching").removeClass("hidden");
-            //$("#client-available").addClass("hidden");
-            //$("#client-unavailable").addClass("hidden");
-
-            // TODO: Should we shutdown and try again? - needs testing
-            //check_ws_socket_timeout = setTimeout(function () {
-            //    find_client();
-            //}, 3000);
+            // Only display a message on the first attempt
```

Comment:
Attempt at removing the single WebSocket connection error that always shows when the page is displayed - unsuccessful (no change in browser behavior), but still a step toward fixing it.

Reply:
The code that creates the websocket does not appear to be enclosed in a try...catch block, so any exceptions thrown by the websocket, such as SecurityError,  are not captured or addressed.

Reply:
I was hoping to find a solution other than try/catch, but I think you're right, that's probably the only thing that will work here.